File: backend/api_arcadia/app/Models/Domicilio.php
<?php

namespace App\Models;

// use Illuminate\Contracts\Auth\MustVerifyEmail;
use Illuminate\Database\Eloquent\Factories\BelongsToRelationship;
use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Relations\BelongsTo;
use Illuminate\Foundation\Auth\User as Authenticatable;
use Illuminate\Notifications\Notifiable;
use Tymon\JWTAuth\Contracts\JWTSubject;

class Domicilio extends Model
{
    protected $fillable = [
        'direccion',
        'piso',
        'puerta',
        'provincia',
        'localidad',
        'cod_postal',
        'id_usuario'
    ];

    public function usuario(): BelongsTo{
        return $this->belongsTo(Usuario::class, 'id_usuario');
    }
}

File: frontend/bibliotecaArcadia/bibliotecaArcadia/10 - Modelos/Domicilio.swift
//
//  File.swift
//  bibliotecaArcadia
//
//  Created by Sandra MZ on 13/4/25.
//

import Foundation

struct ResDomicilio: Decodable {
    let success: Int
    let message: String
    let address: Domicilio?
}
struct Domicilio: Decodable {
    let id: Int
    let id_usuario: Int
    let direccion: String
    let piso: String?
    let puerta: String?
    let provincia: String
    let localidad: String
    let cod_postal: String
}
